README.md

```diff
@@ -85,6 +85,11 @@ python manage.py migrate
 python manage.py runserver
 ```
 
+Load data from fixtures:
+```
+python manage.py loaddata apps\products_catalogue\fixtures\fixtures_products.json
```

Comment:
Actually, at least on Mac, it works only with 
`python manage.py loaddata fixtures_products.json`

Could you check if windows is ok too? If so, please update.

Reply:
Yes, it works on windows too. I think Django itself has to search by its structure for this file.

Reply:
It's supposed to search "fixtures" directory in apps. Thanks for change.